TestingScenarios.md

```diff
@@ -1,15 +1,17 @@
 ## Testing Scenarios
-These are the testing scenarios that need to be validated before shipping a new release.  When an automated test is added, please remove it from the below lists.
 
+These are the testing scenarios that need to be validated before shipping a new release.  When an automated test is added, please remove it from the below lists.
 
 ### Widgets
-1. For each widget: user can configure the widget before pinning
-1. For each widget: user can see widget in dashboard after pinning
-1. For each widget that supports customization: can be customized and persist after restart
-1. Widgets have correct data
-1. Widgets are adequately performant
-1. Notifications can be enabled/disabled in Dev Home settings
-1. Notifications are displayed
+
+1. For each widget: user can configure the widget immediately after first pin
+2. For each widget: user can customize (reconfigure) the widget after its been previously configured and pinned

```

Comment:
Nice thing about markdown is you can just use 1. for everything and it will format it correctly in not-raw mode. Useful if you add items in the middle, then you don't have to go renumber everything. I'd leave the 1s. Not sure if the script expects them as well.

Reply:
Updated